### PR TITLE
Weekly digest

### PR DESCRIPTION
**Pull request by dshevtsov**
## Purpose of this pull request

This pull request adds important documentation updates for the passed week.

## Affected DevDocs pages

- https://devdocs.magento.com/whats-new.html

### CODE REVIEW

File: _data/whats-new.yml [hunk 1]
```diff
@@ -4,8 +4,41 @@ description: |
   We exclude from this list proofreading, spelling checks, and all minor updates.
 link: /whats-new.html
 thread: /whatsnew-feed.xml
-updated: Mon Apr 29 09:56:29 2019
+updated: Mon May  6 15:32:53 2019
```
Comment (shrielenee) [inline _data/whats-new.yml]: Looks like there is an extra space between the May and 6.

File: _data/whats-new.yml [hunk 2]
```diff
@@ -4,8 +4,41 @@ description: |
...
+    [Test your component](https://devdocs.magento.com/guides/v2.3/extension-dev-guide/validate/test-module.html)
+  versions: 2.1.x, 2.2.x, 2.3.x
+  type: Major update
+  date: May  2, 2019
+  link: https://github.com/magento/devdocs/pull/4360
+- description: Page Builder migration documentation - Learn how to use the Page Builder
+    migration module to [migrate your BlueFoot content to Page Builder](https://devdocs.magento.com/page-builder-migration/docs/).
+  versions: 2.3.1
+  type: New topic
+  date: May  2, 2019
```
Comment (shrielenee) [inline _data/whats-new.yml]: Is there an extra space?

**Comment by dshevtsov** [comment]
running tests

File: _data/whats-new.yml (hunk 2 above)
Comment (dshevtsov) [reply _data/whats-new.yml]: yes, it is automatically generated by date converter

File: _data/whats-new.yml (hunk 1 above)
Comment (dshevtsov) [reply _data/whats-new.yml]: yes, it is automatically generated by date converter

Comment (dshevtsov) [reply _data/whats-new.yml]: fixed this in the generator https://github.com/dshevtsov/whats-new-on-devdocs/commit/4022a933dcc85b46e7ba5308fdc10cb88cc3b3e9

File: _data/whats-new.yml (hunk 2 above)
Comment (dshevtsov) [reply _data/whats-new.yml]: fixed this in the generator dshevtsov/whats-new-on-devdocs@4022a93

**Pull request merged**